universe/ObjectMap.cpp

```diff
@@ -136,6 +136,7 @@ namespace {
             // shouldn't be possible to instantiate with this block enabled, but if so,
             // try to generate some useful compiler error messages to indicate what
             // type MappedObjectType is
+            // TODO: Unused alias?
             using GenerateCompileError = typename MappedObjectType::not_a_member_zi23tg;
```

Comment:
Technically unused anywhere in the code base, but comment above indicates it's still needed?

Reply:
"needed" is debatable, but the comment explains the purpose

Reply:
I agree. I'm removing the `TODO` as it's unnecessary.